content/applications/inventory_and_mrp/inventory/shipping/setup/sendcloud_shipping.rst

```diff
@@ -27,7 +27,9 @@ needed.
    in the Odoo database.
 
 .. important::
-   Odoo integration of Sendcloud does **not** work on free plans of Sendcloud.
+   Odoo integration of Sendcloud works on free Sendcloud plans *only* if a bank account is linked,
+   since Sendcloud won't ship for free. To use shipping rules or individual custom carrier
+   contacts, a paid plan of Sendcloud is required.
```

Comment:
Line break a little later on 31
```suggestion
   Odoo integration of Sendcloud works on free Sendcloud plans *only* if a bank account is linked,
   since Sendcloud won't ship for free. To use shipping rules or individual custom carrier contacts,
   a paid plan of Sendcloud is required.
```